public/robots.txt

```diff
@@ -1,2 +1,12 @@
 User-agent: *
 Disallow: /cart
```

Comment:
Your comment seems to indicate that we should add NoIndex here.

Reply:
Updated the content. Google doesn't even support noindex anymore